adafruit_stmpe610.py

```diff
@@ -419,14 +430,14 @@ class Adafruit_STMPE610_SPI(Adafruit_STMPE610):
 
     def __init__(  # pylint: disable=too-many-arguments
         self,
-        spi,
-        cs,
-        baudrate=1000000,
-        calibration=None,
-        size=None,
-        disp_rotation=0,
-        touch_flip=(False, False),
-    ):
+        spi: SPI,
+        cs: Pin,
+        baudrate: int = 1000000,
+        calibration: Optional[Tuple[int, int]] = None,
```

Comment:
This still needs to be addressed similar to the other `calibration` argument.

Reply:
yep - missed that one myself